src/com/epam/university/Group.java

```diff
@@ -0,0 +1,133 @@
+package com.epam.university;
+
+import java.util.Arrays;
+
+public class Group
+{
+    private Student[] studentsInGroup=new Student[3];
+    private String groupName;
+    static String[] courses = new String[] {"Accounting", "Math", "English", "Art","History", "Marketing", "Music", "Literature"};
+
+    public Group(String name)
+    {
+        groupName=name;
+    }
+
+
+    public void setStudentsInGroup(Student[] studentsInGroup) {
+        this.studentsInGroup = studentsInGroup;
+    }
+
+    public Student[] getStudentsInGroup() {
+        return studentsInGroup;
+    }
+
+
+    public void setGroupName(String groupName) {
+        this.groupName = groupName;
+    }
+
+    public String getGroupName() {
+        return groupName;
+    }
+
+
+    //Get average grade of every subject in a particular group
+
+    public static double[] averageOfSubjectsInGroup(Group group)
```

Comment:
This method should not work with hardcoded values. The subjects should be initialized outside of main method and not in Group class, so there will be no need to store all this local variables.